tests/v2/actions/workloads/pods/pods.go

```diff
@@ -66,3 +83,127 @@ func NewPodTemplateWithConfig(secretName, configMapName string, useEnvVars, useV
 	containers := []corev1.Container{container}
 	return workloads.NewPodTemplate(containers, volumes, nil, nil, nil)
 }
+
+// CheckPodLogsForErrors is a helper to check pod logs for errors
+func CheckPodLogsForErrors(client *rancher.Client, clusterID string, podName string, namespace string, errorPattern string, startTime time.Time) error {
```

Comment:
net new in 2.7 vs. 2.8/2.9/main. Can you confirm this isn't an extension on 2.7 for some reason?

Reply:
Hi, this function is on 2.8/2.9/main, but it is just used on 2.9/main, I don't know why it is not an extension on 2.7, I'll remove this function